Server OAPIF: add datetime and properties docs (#4364)
qgis/QGIS-Documentation: source/docs/user_manual/working_with_ogc/server/services.rst

Feature filtering
--------------------

The features available in a collection can be filtered/searched by specifying one or more filters.

Date and time filter
^^^^^^^^^^^^^^^^^^^^

> Collections with date and/or datetime attributes can be filtered by specifying a ``datetime`` argument in the query string.

Reviewer: ```suggestion
Collections with date and/or datetime attributes can be filtered by specifying a ``datetime`` argument
in the query string.
```

> By default the first date/datetime field is used for filtering, this behavior can be configured by settings a "Date" or "Time"

Reviewer 2: ```suggestion
By default the first date/datetime field is used for filtering. This behavior
can be configured by setting a "Date" or "Time"
```

> By default the first date/datetime field is used for filtering, this behavior can be configured by settings a "Date" or "Time"

Reviewer: ```suggestion
By default the first date/datetime field is used for filtering, this behavior can be
configured by setting a "Date" or "Time" dimension in the
:menuselection:`QGIS Server --> Dimension` section of the layer properties dialog.
```

> dimension in the :menuselection:`QGIS Server --> Dimension` section of the layer properties dialog.

Reviewer: ```suggestion
```

> The date and time filtering syntax is fully described in the :ref:`ogc_api_features_api_definition` and

Reviewer: ```suggestion
The date and time filtering syntax is fully described in the
:ref:`ogc_api_features_api_definition` and also supports ranges (begin and end values
are included) in addition to single values.
```

> also supports ranges (begin and end values are included) in addition to single values.

Reviewer: ```suggestion
```

URL examples:

> returns only the the features with date dimension matching ``2019-01-01``

Reviewer 2: ```suggestion
returns only the features with date dimension matching ``2019-01-01``
```

> returns only the the features with date dimension matching ``2019-01-01``

Reviewer: ```suggestion
returns only the the features which have a date and time dimension matching ``2019-01-01``*
```

.. code-block:: none

    http://localhost/qgis_server/wfs3/collection_one/items.json?datetime=2019-01-01

> returns only the the features with datetime dimension matching ``2019-01-01T01:01:01``

Reviewer 2: ```suggestion
returns only the features with datetime dimension matching ``2019-01-01T01:01:01``
```

.. code-block:: none

    http://localhost/qgis_server/wfs3/collection_one/items.json?datetime=2019-01-01T01:01:01

> returns only the the features with datetime dimension in the range ``2019-01-01T01:01:01`` ``2019-01-01T10:10:10``

Reviewer 2: ```suggestion
returns only the features with datetime dimension in the range ``2019-01-01T01:01:01`` - ``2019-01-01T12:00:00``
```
Changed the hour just because the difference at first reading is not obvious

.. code-block:: none

>     http://localhost/qgis_server/wfs3/collection_one/items.json?datetime=2019-01-01T01:01:01/2019-01-01T10:10:10

Reviewer 2: ```suggestion
    http://localhost/qgis_server/wfs3/collection_one/items.json?datetime=2019-01-01T01:01:01/2019-01-01T12:00:00
```

    http://localhost/qgis_server/wfs3/collection_one/items.json?attribute_one=*value

Attribute selection
-------------------

> The features attributes returned by a :ref:`ogc_api_features_features_list` call can be narrowed

Reviewer: ```suggestion
The feature attributes returned by a :ref:`ogc_api_features_features_list` call can be limited
```
